css-canvas-medical/index.html

```diff
@@ -0,0 +1,140 @@
+        the work you do. Capture meaningful data, identify gaps in care, and move seamlessly between tasks within the platform.
+        You stay present with patients while your
+        whole care team stavs up-to-speed. Be a better doctor with Canvas. designed for primary care providers with the help of
+        primary care providers.
+
+      </p>
+      <ul>
+        <li><a href="./get-started.html" class="get">Get Started</a></li>
+      </ul>
+    </div>
+  </section>
+  <section class="middle">
+    <div class="container">
+      <div class="row">
+        <div class="col-1">
+          <h2>Clinicians Prefer Canvas</h2>
+          <p></p>
+          <p class="grey-text">
+            Physician-owned primary care practices across the country have
+          </p>
+          <p class="grey-text" id="middle-text">
+            already made Canvas the fabric of their care.
+          </p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-2">
+          <p class="location">
+            Cascade Family Practice, Portland OR
+          </p>
+          <p class="location">
+            Temescal Creek Medicine, Oakland CA
+          </p>
+          <p class="location">
+            San Juan Healthcare, Friday Harbor WA
+          </p>
+        </div>
+        <div class="col-3">
+          <p class="location">
+            Ward Medical Clinic, Bonifay FL
+          </p>
+          <p class="location">
+            Caribou Medical, Towanda PA
+          </p>
+          <p class="location">
+            Bay Tree Family Care, Red Bay AL
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="bottom">
+    <div class="container-2">
+      <div class="row-2">
+        <div class="col-4">
+          <h2 id="ready">Ready to learn more?</h2>
+          <p class="bottom-text">
+            Schedule a private meeting now
+          </p>
+        </div>
+        <div class="col-5">
+          <ul>
+            <li><a href="./get-started.html" class="get" id="bottom-get">Get Started</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </section>
+  <footer>
+    <div class="container-3">
+      <div class="row-3">
+        <div class="col-6">
+          <img src="images/logo-white.png" alt="white-logo" id="white-logo">
+          <p class="address">
+            2307 Irving Street, Suite 228
+            San Francisco CA 94122
+          </p>
+          <p class="address">
+            (800) 370-1416
+            info@canvasmedical.com
+          </p>
+        </div>
+        <div class="col-7">
+          <h3>Company</h3>
+          <a href="./about-us.html" id="about">About</a>
+          <a href="./about-us.html" id="us">us</a>
+          <a href="./blog.html" id="blog-1">Blog</a>
+        </div>
+        <div class="col-8">
+          <h3 id="prod">Product</h3>
+          <a href="./index.html" id="pro">Product</a>
```

Comment:
I would try to avoid shortening your id names like this. It makes it super hard to read. If I was a new developer trying to take over this project, I would have no idea what "pro" is. I would also consider if you need to use these id's at all. If you only gave these an id so you could apply styling to them, I would try and use other methods first, like using direct child `<` or `nth-child()`